Alexey/Text/text_editor/tests/test_text_editor.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+from Alexey.Text.text_editor.main.text_editor import *
+
+
+class TestTextEditor(unittest.TestCase):
+    def test_write_normal_text_and_expect_file_with_text(self):
+        expect_set = "Awesome!"
+        some_text = "Awesome!"
+        write_file(some_text)
+        self.assertEqual(expect_set, check_file())
+
+    def test_append_normal_text_and_expect_file_with_appended_text(self):
+        expect_set = "Awesome!Awesome!"
+        some_text = "Awesome!"
+        edit_file(some_text)
+        self.assertEqual(expect_set, check_file())
+
+    def test_send_wrong_type_of_data_to_write_function_and_expect_type_error(self):
+        with self.assertRaises(TypeError) as context:
```

Comment:
If exception is not expected, tests can be ignored for exceptions 
If you remove exception check, please leave test for int value